job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/triggers/PullRequestBuilderContext.groovy

```diff
@@ -90,4 +99,28 @@ class PullRequestBuilderContext extends AbstractContext {
     void allowMembersOfWhitelistedOrgsAsAdmin(boolean allowMembersOfWhitelistedOrgsAsAdmin = true) {
         this.allowMembersOfWhitelistedOrgsAsAdmin = allowMembersOfWhitelistedOrgsAsAdmin
     }
+
+    /**
+    * @since 1.22
+    */
+    @RequiresPlugin(id = 'ghprb', minimumVersion = '1.22-0')
+    void commitStatusContext(String commitStatus) {
+        this.commitStatusContext = commitStatus
+    }
+
+    void triggeredStatus(String triggeredStatus) {
```

Comment:
Add a `@RequiresPlugin` annotation with the correct minimum version for all new methods.